Assets/Scripts/GameLogic/MathQuestionGenerator.cs

```diff
@@ -110,6 +116,26 @@ void Start()
         a1.text = randomButton == 1 ? answer + "" : wrongAnswers.Pop() + "";
         a2.text = randomButton == 2 ? answer + "" : wrongAnswers.Pop() + "";
         a3.text = randomButton == 3 ? answer + "" : wrongAnswers.Pop() + "";
+        
+        // Start timer
+        remainingTime = 10.00f;
+        StartCoroutine(RunTimer());
+    }
+
+    private IEnumerator RunTimer()
+    {
+        while (remainingTime >= 0.005f) 
+        {
+            remainingTime -= Time.deltaTime;
+            if (remainingTime <= 5.0f) timer.color = Color.red;
+            timer.text = "TIME: " + remainingTime.ToString("0.00");
```

Comment:
c-c-c-c-c-c :)

Reply:
This will only ever occur in one spot so I said fk it